api/tests/Api/ActivityProgressLabel/UpdateActivityProgressLabelTest.php

```diff
@@ -0,0 +1,149 @@
+            ->request('PATCH', '/activity_progress_labels/'.$activityProgressLabel->getId(), ['json' => [
+                'position' => 3,
+                'title' => 'NewTitle',
+            ], 'headers' => ['Content-Type' => 'application/merge-patch+json']])
+        ;
+        $this->assertResponseStatusCodeSame(200);
+        $this->assertJsonContains([
+            'position' => 3,
+            'title' => 'NewTitle',
+            '_links' => [
+                'camp' => [
+                    'href' => $this->getIriFor('camp1'),
+                ],
+            ],
+        ]);
+    }
+
+    public function testPatchActivityProgressLabelIsAllowedForManager() {
+        /** @var ActivityProgressLabel $activityProgressLabel */
+        $activityProgressLabel = static::$fixtures['activityProgressLabel1'];
+        static::createClientWithCredentials()
+            ->request('PATCH', '/activity_progress_labels/'.$activityProgressLabel->getId(), ['json' => [
+                'position' => 3,
+                'title' => 'NewTitle',
+            ], 'headers' => ['Content-Type' => 'application/merge-patch+json']])
+        ;
+        $this->assertResponseStatusCodeSame(200);
+        $this->assertJsonContains([
+            'position' => 3,
+            'title' => 'NewTitle',
+            '_links' => [
+                'camp' => [
+                    'href' => $this->getIriFor('camp1'),
+                ],
+            ],
+        ]);
+    }
+
+    public function testPatchActivityProgressLabelDisallowsChangingCamp() {
+        /** @var ActivityProgressLabel $activityProgressLabel */
+        $activityProgressLabel = static::$fixtures['activityProgressLabel1'];
+        static::createClientWithCredentials()
+            ->request('PATCH', '/activity_progress_labels/'.$activityProgressLabel->getId(), ['json' => [
+                'camp' => $this->getIriFor('camp2'),
+            ], 'headers' => ['Content-Type' => 'application/merge-patch+json']])
+        ;
+        $this->assertResponseStatusCodeSame(400);
+        $this->assertJsonContains([
+            'detail' => 'Extra attributes are not allowed ("camp" is unknown).',
+        ]);
+    }
+
+    public function testPatchActivityProgressLabelValidatesBlankTitle() {
+        /** @var ActivityProgressLabel $activityProgressLabel */
+        $activityProgressLabel = static::$fixtures['activityProgressLabel1'];
+        static::createClientWithCredentials()
+            ->request('PATCH', '/activity_progress_labels/'.$activityProgressLabel->getId(), ['json' => [
+                'title' => '',
+            ], 'headers' => ['Content-Type' => 'application/merge-patch+json']])
+        ;
+        $this->assertResponseStatusCodeSame(422);
+        $this->assertJsonContains([
+            'violations' => [
+                [
+                    'propertyPath' => 'title',
+                    'message' => 'This value should not be blank.',
+                ],
+            ],
+        ]);
+    }
```

Comment:
either use Gedmo\Sortable or verify the unique constraint on position

Optionally:
\+ test trimming of title
\+ test length of title is validated